README.md

```diff
@@ -8,99 +8,166 @@ _user_ not a _developer_.
-seen weather reports, beautiful colors, computer temperatures in prompts. At
-heart, though, they they all carry your command to the operating system and run
-it.
+First of all, we have to be honest here: the difference between "shell" and
+"terminal" is quite easy to get confused by. To make things worse, developers
+are often imprecise and say one to mean the other and vice versa.
 
-You can ask the computer to do many types of tasks. From the command prompt you
-can move or rename files just like you would do from a window environment.  You
+In order to try to help build your instinct for understanding the concepts,
+we'll often "over-explain" or make a sentence seem ridiculously long. We're
+doing this intentionally in order to try to help you build an intuition about
+the actors in this story.
+
+Also, to help keep terminology from getting in the way, we involve genies.
+We admit, this is a whimsical approach but assure you no genies were harmed in
+the production of this lesson.
+
+Be patient with yourself as you learn this material.
+
+### Terminal Programs
+
+A terminal program is a program that listens for keyboard input and hands it
+off to a magical genie who lives inside your computer. Also, when the magical
+genie tells the terminal to draw letters on its screen, it does so. For
+example, when you are in a terminal program and type the letter `l` the
+terminal program senses the `l` and tells the genie. The genie then tells the
+terminal to draw an `l` on its screen. The terminal then looks up your font
+preferences and color preferences and draws the appropriate `l`.
+
+This might be confusing, but let's compare it to television. A television is a
+device that listens to your remote control and then tells a magical genie
+(different genie, of course) which feed of data to find. When the genie finds
+the data (maybe on a coaxial cable, or satellite dish) it commands the
+television to display the picture. On most modern televisions, in fact, the
+magical genie tells the television to draw a picture 120 times per second!
+
+The essential fact is that a terminal is a means for talking to the operating
+system's genie.  In lessons we'll refer to the "terminal" as the thing you type
+in, as a program you can launch, as a space you can click in.
+
+## Recognize a Shell Interface
+
+### Shell Interfaces Described
+
+OK, sorry, genies don't exist. Well, at the very least, operating system genies
+don't exist.
+
+Above, when we said that the terminal hands requests and receives commands from
+a genie, a thing that _thinks_ and thing that _can find out things_, we were
+actually talking about "the shell." The _shell_ knows how to ask the operating
+system how big the hard drive is. The _shell_ knows how to ask the operating
+system if the web site `flatironschool.com` can be reached. The _shell_ **does
+not** know how to display "Hello World" in 16 point Monaco font on a black
+background, that's the _terminal_.
+
```

Comment:
Ok, bringing it back around here. I think it can definitely still use a little tweaking, but it's way better here.